fe/fe-core/src/main/java/com/starrocks/qe/StmtExecutor.java

```diff
@@ -905,6 +908,10 @@ private boolean tryProcessProfileAsync(ExecPlan plan, int retryIndex) {
             }
             QeProcessorImpl.INSTANCE.unMonitorQuery(executionId);
             QeProcessorImpl.INSTANCE.unregisterQuery(executionId);
+            if (Config.enable_collect_query_detail_info && Config.enable_profile_log) {
+                String jsonString = GSON.toJson(queryDetail);
+                PROFILE_LOG.info(jsonString);
```

Comment:
Is profile_log a disk file? if so,  is the profiles in this file  readable?  if it need not to be readable, can we compress profiles to save disk room?

Reply:
this is not used on noraml case, so sass team just need a original profile json text as they said